src/commands/publish-helm.yml

```diff
@@ -0,0 +1,50 @@
+description: >
+  Publish a helm chart
+
+steps:
+- run:
+    name: Publish
+    command: |
+        ## Get chart version and make sure it's an updated version
+
+        CHART_VERSION=$(yq r Chart.yaml version)
+
+        if [ "$CHART_VERSION" == "null" ]; then
+            echo "Could not find chart version"
+            exit 1
+        fi
+
+        set +e # allow grep to fail
+        git --no-pager tag | grep $CHART_VERSION
+
+        if [ $? == 0 ]; then
+            echo "Chart with version ${CHART_VERSION} has already been published"
+            exit 0
+        fi
+
+        set +e
+
+        ## Push new tag to repo
+
+        git remote rm origin
+        git remote add origin "https://pennappslabs:${CR_TOKEN}@github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}.git"
```

Comment:
I'm not a huge fan of using a username and password here. I'd rather use ssh, but that also creates a bunch of problems. I think it may make the most sense to require the commit to be tagged already and remove this block of logic

Reply:
Also git user and email should be able to be set with orb parameters

Reply:
I disagree about the tagging. Without automating tagging w/ circleci, someone with admin on this repo has to manually git tag each time. This automates it and ensures you never accidentally deploy the same tag twice.
Also CR_TOKEN already has to be provided for chart-releaser to work. I think if we already have one method of git auth in there, we shouldn't put in a second.